scripts/integration/integration.py

```diff
@@ -486,14 +497,39 @@ def main():
         default='scripts/flamegraph',
         type=Path,
         help='Path to flamegraph script')
+    parser.add_argument(
+        '-v',
+        '--verbosity',
+        default='DEBUG',
+        help='Set the logging verbosity')
 
     args = parser.parse_args()
+
+    # Setup logging.
+    LOGGER.setLevel(logging.DEBUG)
```

Comment:
It is not clear how this setter interacts with the one on the `StreamHandler` or which one takes precedence. Also should this not be `args.verbosity`?

Reply:
This verbosity level is the global level. It could certainly be `args.verbosity`. My understanding is that the individual stream handlers have their own filtering that precedes the filtering of the global log level.